test/transpile/lualib.spec.ts

```diff
@@ -29,3 +30,39 @@ test("Lualib bundle does not assign globals", () => {
         .withLanguageExtensions()
         .expectNoExecutionError();
 });
+
+test("Lualib bundle recompile", () => {
+    const { transpiledFiles } = util.testExpression`Array.isArray({})`
+        .setOptions({
+            recompileLuaLib: true,
+            luaPlugins: [{ name: path.join(__dirname, "./plugins/beforeEmit.ts") }],
+        })
+        .expectToHaveNoDiagnostics()
+        .getLuaResult();
+    const lualubBundle = transpiledFiles.find(f => path.basename(f.outPath) === "lualib_bundle.lua");
+    expect(lualubBundle?.lua).toContain(
+        "-- Comment added by beforeEmit plugin\n-- Comment added by beforeEmit plugin\n"
+    );
+});
+
+test("Lualib require minimal recompile", () => {
+    util.testExpression`Array.isArray(123)`
+        .setOptions({
+            luaLibImport: LuaLibImportKind.RequireMinimal,
+            recompileLuaLib: true,
+            luaPlugins: [{ name: path.join(__dirname, "./plugins/visitor.ts") }],
+        })
+        .expectToHaveNoDiagnostics()
```

Comment:
Could you add an assertion that the lualib file (s) are output at the expected location?